lib/ffi/Cargo.toml

```diff
@@ -9,7 +9,7 @@ crate-type = ["cdylib"]
 [dependencies]
 vandals-and-heroes = { path = "../.." }
 raw-window-handle = "0.6.2"
-log = "0.4.22"
+log = "0.4"
```

Comment:
do we still need the FFI? I was hoping you'd be comfortable with Rust modules by now